conditional/blueprints/major_project_submission.py

```diff
@@ -61,6 +69,9 @@ def submit_major_project(user_dict=None):
         return jsonify({"success": False}), 400
     project = MajorProject(user_dict['username'], name, description)
 
+    username = user_dict['username']
+    send_slack_ping({"text":f"<!subteam^S5XENJJAH> *{get_member_name(username)}* ({username})\n"
+                            f" submitted their major project, *{name}*!"})
```

Comment:
Why are these two separate strings? Also, why newline between name and the rest of the text?

This is what it looks like on Slack
<img width="536" alt="image" src="https://user-images.githubusercontent.com/40615740/189137557-585662d2-908b-4bbb-afa4-a7a833be6c3f.png">
Ignore the fact that I removed the ping